httpproxy.py

```diff
@@ -139,8 +139,12 @@ def send_archived_http_response(self, response):
                       self.server.traffic_shaping_delay_ms)
         time.sleep(self.server.traffic_shaping_delay_ms / 1000.0)
       if is_replay and self.server.use_delays:
-        logging.debug('Using delays (ms): %s', response.delays)
-        time.sleep(response.delays['headers'] / 1000.0)
+        if self.server.use_connect_delays:
```

Comment:
I think it would be a little clearer as such:
```
logging.debug('Using delays from headers (ms): %s', response.delays['headers'])
latency = response.delays['headers']
if self.server.use_connect_delays:
  logging.debug('Using delays from connect (ms): %s', response.delays['connect'])
  latency += response.delays['connect']
time.sleep(latency / 1000.0)
```

Reply:
Updated